requirements.txt

```diff
@@ -6,6 +6,7 @@ django-cors-headers==3.0.2       # allows CORS requests for client-side developm
 social-auth-app-django==4.0.0    # the package for Django to authenticate users with social medieas
 social-auth-core==3.3.3          # the Python social authentication package. Required by social-auth-app-django
 google-auth==1.22.1              # for Google service account client and secure sessions using access tokens
+rsa==4.6                         # required by google-auth
```

Comment:
to be clear: the issue synk is reporting is that this is a downstream dependency of the google auth package you added and it is very vulnerable and unsafe to use. Adding the vulnerable downstream dependency explicitly does absolutely nothing to make this less of a security threat. We need to either a) remove the package that introduces vulnerability (which is google-auth) or b) document why it is absolutely essential that we use a vulnerable package. I d==o not think we need the google auth package. It is a security threat. We can use something else that isn't a security threat. Take it out